src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Utils/SetupUtil.cs

```diff
@@ -56,7 +56,7 @@ public static class SetupUtil
 
                     services.AddSingleton<Altinn.Common.PEP.Interfaces.IPDP, PepWithPDPAuthorizationMockSI>();
 
-                    services.AddSingleton<IValidation, ValidationAppSI>();
+                    services.AddScoped<IValidation, ValidationAppSI>();
```

Comment:
`IValidation` is being registered as Transient during startup of the app, we can do that here for the tests as well.

Reply:
fixed